electricitylci/data/International_Electricity_Mix.csv

```diff
@@ -0,0 +1,51 @@
+Subregion,FuelCategory,Electricity,NERC,Generation_Ratio

```

Comment:
I think it would be best to make clear that this is the mix for 2016 (presumably). This could be as simple as adding a column with the year, so that we can just add rows for future years if we need to.

Reply:
I assume the NERC column is needed for the aggregation. I think there's room to consolidate this with (canadian_imports.csv)[https://github.com/USEPA/ElectricityLCI/blob/master/electricitylci/data/canadian_imports.csv]. If we can get rid of the NERC column, then it just simplifies the file a bit.

Reply:
@jump2conclusionsmatt , where/how is that other canadian_imports.csv used? We saw that, but it was an incomplete list of provinces and had a different naming convention than has been utilized for the NERC trading previously.

Reply:
@TJTapajyoti , I think we can remove the NERC column. We had only included that to check in the excel sheet prior to integration with the repo. The NERC-level supply pools should identify the correct provinces to draw from independent from this table, right?

Reply:
> @jump2conclusionsmatt , where/how is that other canadian_imports.csv used? We saw that, but it was an incomplete list of provinces and had a different naming convention than has been utilized for the NERC trading previously.

It's used in `import_impacts.generate_canadian_mixes()`. I don't think we need to consolidate now, but it should be done in the next revision. I think given what I've seen here, I'd lean more towards implementing this version of Canadian mixes (connecting us average profiles within openLCA), but that's going to require adding the ability to create the US average mixes from data other than eGRID.  I added a separate issue to track this.

Reply:
@hottleta @jump2conclusionsmatt I agree. I looked at the Canadian_imports.csv file. If the names are changed to proper full form, we can use this one to create Canadian mixes. However, mexico is missing from this file. So we need to add it in. But, yes, in next versions, please feel free to replace the International_Electricity_mix.csv file. I will add the Year column and remove NERC region. The NERC regions are useless. Its not used in calculations.